### ISSUE TITLE
event.az and event.simulation_truth.air_shower.phi use different coordinate systems

### ISSUE THREAD

**Issue by maxnoe** [fact-project/photon_stream#71]
One is Corsika (0° at megnetic north), one is Ceres (0° at geographic south)

**Comment by relleums** [comment]
Indeed. I have code which solves this: https://github.com/relleums/phs_air_shower_feature_generation
But where should it go? Any ideas? I discussed this also lately with Kai.

**Comment by maxnoe** [comment]
I think we should just fix it. Use the astropy's and FACT-Tools coordinate system definition.

You can directly do this by not using `MSimPointingPos` or the data from the corsika file but use `source_position_{az,zd}` and `pointing_position_{az,zd}` with the current FACT-Tools version.

I think we should do this after we released FACT-Tools 1.0. We plan a complete reprocessing of all data with FACT-Tools 1.0

**Comment by maxnoe** [comment]
What's even worse is that `event.az` is in a different coordinate system for observations and simulations
